pages/fixit/onboarding.md

```diff
@@ -0,0 +1,41 @@
+---
+permalink: /fixit/onboarding
+---
+# Onboarding Fixit
+
+The theme for 18F's first official [fixit](/fixit) is onboarding, and it will culminate on Friday, April 17th.
+
+The ops and onboarding teams are doing tremendous work to make sure new hires have the best possible transition to life at 18F.
+
+There's additional work to do, however, and a key component involves making sure our individual teams and working groups are ready to absorb the upcoming influx of new hires. Also, as the Hub becomes integral to onboarding, we need to make sure the information there is correct and useful.
+
+The onboarding process is more than documentation, but documentation is critical to making sure onboarding can scale.
+
+## The Plan
+
+A large cohort is starting at 18F on April 6th, and these new teammates are the incentive for the onboarding fixit. Here's an outline of our thinking.
+
+### Before April 6th
+
+Project teams and working groups get their onboarding houses in order as much as possible. For example:
+
+* Is your project or group on the 18F dashboard?
+* Do you have a getting started checklist? What access, training, and accounts would someone need to join your team?
+* Is there documentation for setting up a dev environment?
+* Who adds new folks to recurring meeting invites?
+
+
+### April 6th - April 16th
+
+Our new coworkers onboard using the materials we've created. We all use this opportunity to record opportunities for improvement. The specific feedback mechanisms will likely be project-specific (*e.g.*, GitHub issue, Trello card). In addition, fixit organizers will survey the new cohort to get additional ideas.
+
+### Fixit Day: April 17th
+
+We'll congregate to tackle the collected issues (and create new ones if necessary), reporting along the way. We'll likely use a Google form to track what gets done. This tracking is a critical step, since we want to collect fixit metrics.
+
+### Post-Fixit
+
+The fixit organizers will survey participants, sift through the collected metrics, and report back:
+
+* How many issues did we fix?
```

Comment:
- How many people participated?
- What did the fixit help accomplish, for individuals, projects, working groups and guilds?
- What is now possible after the fixit that wasn't possible before?